pages/common/d8.md

```diff
@@ -0,0 +1,16 @@
+# d8
+
+> Developer shell for the V8 JavaScript engine.
+> More information: <https://v8.dev/docs/d8>.
+
+- Start a REPL (interactive shell):
+
+`d8`
+
+- Run an ECMAScript file:
```

Comment:
```suggestion
- Run a JavaScript file:
```

Reply:
Why?

Reply:
Why `ECMAScript` if this works with `JavaScript`?

Reply:
It's true that `ECMAScript` is the proper name of the Javascript language in the specification itself, but most people will recognise it as Javascript.

Reply:
V8 also uses both names, they literally have both in the same paragraph: "V8 is Google’s open source high-performance JavaScript and WebAssembly engine, written in C++. It is used in Chrome and in Node.js, among others. It implements ECMAScript and WebAssembly, and runs on [...]"

Reply:
Ah, I see. It says that it _implements_ ECMAScript (the standard), but we generally refer to JavaScript files instead of ECMAScript files given the file extension (e.g. .js / .mjs).